docs/api.md

```diff
@@ -1951,6 +2008,39 @@ You can fully delete a link by issuing `DELETE` to its resource. This will remov
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+
+## Dataset Properties [/v1]
+
+_(introduced: version 2022.3)_
+
+Datasets are created and updated through Forms. Dataset-related Forms follow a new version of the [ODK XForms specification](https://getodk.github.io/xforms-spec) that allow them to define a Dataset and a mapping of Form Fields to Dataset Properties. Submissions from such a Form can create Entities within the Dataset defined in the Form. 
+
+Currently, Datasets and Dataset Properties are purely additive. Multiple Forms can add Properties to the same Dataset and multiple Forms can create Entities in the same Dataset. Not all Properties of a Dataset have to be included in a Form for that Dataset. For example, one Form publishing to a Dataset called `trees` could add `location` and `species`, while another could add `species` and `circumference`. The Properties of the Dataset would be the union of Properties from all Forms for that Dataset (`location`, `species`, `circumference`).
```

Comment:
This all feels like it should be in user docs instead. But I think it's ok to leave and keep iterating on.